src/MicrodataDOMElement.php

```diff
@@ -160,9 +163,9 @@ protected function getChildElementNodes()
      *
      * @param string $attributeName Name of the attribute
      *
-     * @return array|array[]|false|string[]
+     * @return array|bool
```

Comment:
Type hints don't support false, but we can at least specify it in DocBlocks.

```suggestion
     * @return array|false
```